docs/index.md

```diff
@@ -0,0 +1,24 @@
+Hyperbahn
+=========
+
+Hyperbahn is a service discovery and routing network that allows simple, reliable
+communication with any service without any knowledge of where that service runs.
+
+Hyperbahn supports the TChannel RPC protocol and works with Ringpop's gossip protocol
+and consistent hash ring topology.
+
+Hyperbahn and TChannel clients are currently supported in Go, Python, and Node.js.
+
+* Go
+    * [Guide](go-guide.md)
+    * [API Documentation](http://godoc.org/github.com/uber/hyperbahn-go)
+
+
+* Node
+    * [Guide](http://hyperbahn-node.readthedocs.org/en/latest/GUIDE/)
+    * [API Documentation](http://hyperbahn-node.readthedocs.org/en/latest/)
```

Comment:
These links don't work yet. You don't think we'd want docs specifically for
Hyperbahn, as its own product?

On Fri, Oct 30, 2015 at 3:56 PM, Jake Verbaten <notifications@github.com>
wrote:

> In docs/index.md
> <https://github.com/uber/hyperbahn/pull/103#discussion_r43560644>:
>
> > +Hyperbahn is a service discovery and routing network that allows simple, reliable
> > +communication with any service without any knowledge of where that service runs.
> > +
> > +Hyperbahn supports the TChannel RPC protocol and works with Ringpop's gossip protocol
> > +and consistent hash ring topology.
> > +
> > +Hyperbahn and TChannel clients are currently supported in Go, Python, and Node.js.
> > +
> > +* Go
> > +    * [Guide](go-guide.md)
> > +    * [API Documentation](http://godoc.org/github.com/uber/hyperbahn-go)
> > +
> > +
> > +* Node
> > +    * [Guide](http://hyperbahn-node.readthedocs.org/en/latest/GUIDE/)
> > +    * [API Documentation](http://hyperbahn-node.readthedocs.org/en/latest/)
>
> do we want to point to "hyperbahn-node" docs or "tchannel-node" docs ?
>
> I feel like we should point to tchannel documentation.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/uber/hyperbahn/pull/103/files#r43560644>.
>